### PR TITLE
Add python version option for deployment

### PR DESCRIPTION
**Pull request by yunti**
Fix for #322

### CODE REVIEW

File: {{cookiecutter.repo_name}}/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,8 @@
+{% if cookiecutter.use_python2= 'n' -%}
```
Comment (luzfcb) [inline {{cookiecutter.repo_name}}/Dockerfile]:
```python
{% if cookiecutter.use_python2 == 'n' -%}
```

File: {{cookiecutter.repo_name}}/runtime.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{% if cookiecutter.use_python2= 'n' -%}
```
Comment (luzfcb) [inline {{cookiecutter.repo_name}}/runtime.txt]:
```python
{% if cookiecutter.use_python2 == 'n' -%}
```

**Comment by yunti** [comment]
Hmm bit of a mess in terms of commits. Just let me know if you would rather I delete and resubmit.

**Comment by crdoconnor** [comment]
Should I amend this too https://github.com/pydanny/cookiecutter-django/pull/340/files#diff-5f8fc5f3de1f2873b3cc21cf3f6df870R21 ?

**Comment by audreyr** [comment]
Awesome. I went over this with @pydanny and am going to merge it in. Thanks @yunti and don't worry about the commits.

We should see about if 3.5 will work in Dockerfile and Heroku runtime rather than 3.4, but that can be a later pull request.

**Comment by yunti** [comment]
Thanks , saw pydanny added 3.5 to docker file.  3.5 build pack isn't in Heroku yet but from this pull request should hopefully be soon: https://github.com/heroku/heroku-buildpack-python/pull/250

**Pull request merged**